state-chain/chains/src/sol/api.rs

```diff
@@ -400,7 +402,10 @@ impl<Environment: SolanaEnvironment> SolanaApi<Environment> {
 		// Get the Address lookup tables. Chainflip's ALT is proceeded with the User's.
 		// TODO roy: Coordinate with Ramiz on the interface for getting ALTS
```

Comment:
What's happening with this comment?
CC: @syan095

Reply:
I see comment is removed here: https://github.com/chainflip-io/chainflip-backend/pull/5667 resolving.